scripts/index.js

```diff
@@ -56,6 +60,12 @@ import 'superdesk-settings';
 import 'superdesk-ingest';
 import 'superdesk-search-providers';
 
-// App
-import 'app/scripts/translations';
-import 'app/main';
+// load spellcheckers
+if (appConfig.spellcheckers.indexOf('tansa') > -1) {
```

Comment:
Comment from @Aca-jov - use `features.tansaProofing` instead of `config.spellcheckers`